.gitignore

```diff
@@ -1,3 +1,4 @@
 phpunit.xml
 vendor
 composer.lock
+.idea/
```

Comment:
Please put this in your global gitconfig.  I'm not a fan of having IDE specific configs in repository gitignore files because developers choose to use a specific IDE.  They should be able to choose any IDE they wish.

https://help.github.com/articles/ignoring-files/#create-a-global-gitignore